src/Umbraco.Cms.Api.Management/ViewModels/ContentType/PropertyTypeContainerViewModelBase.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Umbraco.Cms.Api.Management.ViewModels.ContentType;
+
+public abstract class PropertyTypeContainerViewModelBase
+{
+    public Guid Key { get; set; }
+
+    public Guid? ParentKey { get; set; }
+
+    public string? Name { get; set; }
+
+    // NOTE: This needs to be a string because it can be anything in the future (= not necessarily limited to "tab" or "group")
+    public string Type { get; set; } = string.Empty;
```

Comment:
But is it a breaking change to add a value to an enum in the future?

Reply:
It is not, but the point is we don't want to know about them in the future; they are purely data needed for presentation, we just need to store the data. So sometime down the line we'll get rid of the enum entirely 😄